lib/db/mem.js

```diff
@@ -549,13 +553,18 @@ module.exports = function (log, error) {
     item.accountCreatedAt = account.createdAt
 
     if (unverifiedTokens[id]) {
-      item.mustVerify = unverifiedTokens[id].mustVerify
+      // Coalesce `mustVerify`
+      if (sessionTokens[id].mustVerify) {
+        item.mustVerify = sessionTokens[id].mustVerify
+      } else {
+        item.mustVerify = unverifiedTokens[id].mustVerify
+      }
```

Comment:
This code confused me initially. Given that `item.mustVerify` is already initialised to `sessionTokens[id].mustVerify || null` on line 546, could it be simplified to a simple `if (! item.mustVerify) {` condition?

Reply:
s/simple/single/